services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The type annotations for `genres_ids` and `actors_ids` use `list[int]`, which is only supported in Python 3.9 and later. If you need to support earlier versions, consider using `List[int]` from the `typing` module.